mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1160,7 +1230,7 @@ def _generate_model_predictions(self):
                     error_code=INVALID_PARAMETER_VALUE,
                 )
 
-            self.metrics_dict.update({"latency": pred_latencies})
+            self.metrics_values.update({"latency": MetricValue(scores=pred_latencies)})
```

Comment:
Can we rename `MetricValue` to a name like `MetricSummary`?

Reply:
I think we are already decided on the naming cc @sunishsheth2009

Reply:
If we already did, that's ok. Just thought `MetricValue` sounds a bit confusing to me because it sounds like a number.